mscore/albummanager.ui

```diff
@@ -1,174 +1,230 @@
+          <property name="toolTip">
+           <string>Move current score down in list</string>
+          </property>
+          <property name="text">
+           <string notr="true"/>
+          </property>
+          <property name="icon">
+           <iconset resource="../../../../../../../tmp/QtCreator-xSnvKu/qtc-cmake-bsxObyxX/mscore/musescore_qmlcache.qrc">
+            <normaloff>:/data/icons/arrow_down.svg</normaloff>:/data/icons/arrow_down.svg</iconset>
+          </property>
+         </widget>
+        </item>
+       </layout>
+      </item>
+      <item row="13" column="1">
+       <widget class="QPushButton" name="closeAlbumButton">
+        <property name="text">
+         <string>Close Album</string>
+        </property>
+       </widget>
+      </item>
+      <item row="0" column="1">
+       <layout class="QHBoxLayout" name="horizontalLayout_4">
+        <item>
+         <widget class="QPushButton" name="settingsButton">
+          <property name="sizePolicy">
+           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
+            <horstretch>0</horstretch>
+            <verstretch>0</verstretch>
+           </sizepolicy>
+          </property>
+          <property name="layoutDirection">
+           <enum>Qt::LeftToRight</enum>
+          </property>
+          <property name="text">
+           <string>...</string>
+          </property>
+         </widget>
+        </item>
+       </layout>
+      </item>
+      <item row="1" column="0" rowspan="13">
+       <widget class="QTableWidget" name="scoreList">
+        <property name="dragEnabled">
+         <bool>true</bool>
+        </property>
+        <property name="dragDropOverwriteMode">
+         <bool>false</bool>
+        </property>
+        <property name="dragDropMode">
+         <enum>QAbstractItemView::NoDragDrop</enum>
+        </property>
+        <property name="defaultDropAction">
+         <enum>Qt::MoveAction</enum>
+        </property>
+        <property name="showGrid">
+         <bool>false</bool>
+        </property>
+        <property name="columnCount">
+         <number>2</number>
+        </property>
+        <column>
+         <property name="text">
+          <string>Title</string>
+         </property>
+        </column>
+        <column>
+         <property name="text">
+          <string>Duration</string>
+         </property>
+        </column>
+       </widget>
+      </item>
+     </layout>
+    </item>
+   </layout>
+  </widget>
  </widget>
- <tabstops>
-  <tabstop>scoreList</tabstop>
-  <tabstop>up</tabstop>
-  <tabstop>down</tabstop>
-  <tabstop>add</tabstop>
-  <tabstop>addNew</tabstop>
-  <tabstop>remove</tabstop>
- </tabstops>
  <resources>
-  <include location="musescore.qrc"/>
+  <include location="../../../../../../../tmp/QtCreator-xSnvKu/qtc-cmake-bsxObyxX/mscore/musescore_qmlcache.qrc"/>
+  <include location="../../../../../../../tmp/QtCreator-xSnvKu/qtc-cmake-bsxObyxX/mscore/musescore_qmlcache.qrc"/>
```

Comment:
I know QtCreator tend to do this, but once is enough.
And in this case it should rather stay like before the change